public/scripts/tutorial.js

```diff
@@ -0,0 +1,127 @@
+var AddProduct = React.createClass({
+  handleSubmit:function(e) {
+    e.preventDefault();
+    var cat=React.findDOMNode(this.refs.category).value.trim();
+    var name=React.findDOMNode(this.refs.name).value.trim();
+    var price = React.findDOMNode(this.refs.price).value.trim();
+    var stock = React.findDOMNode(this.refs.stocked).value();
+    if(!cat || !product || !price){
+      return;
+    }
+    this.props.onSubmit({category: cat, price: price, stocked: stock, name: name});
+    React.findDOMNode(this.refs.category).value='';
+    React.findDOMNode(this.refs.name).value='';
+    React.findDOMNode(this.refs.price).value='';
+  },
+  render: function() {
+    return (
+        <div className="addProduct">
+        <h3> Add a product </h3>
+        <form className="commentForm">
```

Comment:
@ericyliu I think @liza8bit is passing onSubmit a [different function](https://github.com/liza8bit/stocklist/pull/1/files#diff-d56dcef1dcd3108eb64737d609d57820R93). @liza8bit maybe try putting a `debugger;` statement [here](https://github.com/liza8bit/stocklist/pull/1/files#diff-d56dcef1dcd3108eb64737d609d57820R95) to figure out why it's not getting called